File: 5_D/src/Bildirim.java
public class Bildirim implements Iletisim {
    
    
    public void bildirimGoster(String IP,String mesaj){
        System.out.println(IP+" adresine "+mesaj+" gonderildi");
    }
    public void bilgiGonder(Kullanici kullanici,String mesaj){
        bildirimGoster(kullanici.getIP(), mesaj);
    }


}

File: SOLID/5_D/python/Bildirim.py
from Iletisim import Iletisim
from Kullanici import Kullanici
from abc import ABC,abstractmethod

class Bildirim(Iletisim):
    
    def bildirimGoster(self,ip,mesaj):
        print(f"{ip} adresine {mesaj} gönderildi")
        

    def bilgiGonder(self,kullanici,mesaj):
        self.bildirimGoster(kullanici.getIp(),mesaj)
     
    
        